src/main/scala/com/codacy/client/bitbucket/client/RequestResponse.scala

```diff
@@ -27,9 +34,9 @@ object RequestResponse {
 
   def apply[A](r1: RequestResponse[Seq[A]], r2: RequestResponse[Seq[A]]): RequestResponse[Seq[A]] = {
```

Comment:
Can we deprecated this, and call it "applyDiscardingPaginationInfo", and make sure that we only use it on the old executePaginatedLoop?